Stripe/STPCustomerContext.m

```diff
@@ -48,6 +49,10 @@ - (void)clearCachedCustomer {
 }
 
 - (void)setCustomer:(STPCustomer *)customer {
+    if (self.includeApplePaySources) {
+        [customer updateSourcesWithResponse:customer.allResponseFields
```

Comment:
This is kind of funky/subtle/non-obvious.

* This mutates the `STPCustomer` object passed into the setter, which the caller still has a reference to.
* I didn't understand why you changed `customer` -> `self.customer` on L92 below.
  * I'm guessing because `setCustomer:` has side effects, but because they're just references to the same object, I don't think that change actually changes the behavior?

I see that this is a private property, and I think there're only three places that call the setter and none of them use the customer afterwords, so it's fine as-is (although maybe a well-placed & well-worded comment could help?).

I *think* it might be easier to understand & maintain if this was (duplicated and) moved up a level in the call stack to those places that receive `STPCustomer` objects from `STPAPI`. In each of them it's more clear that the `STPCustomerContext` completely owns the object & mutating it is fine, as well as not having (potentially) surprising behavior "hidden" in the setter.

Reply:
Ah yeah that's good feedback, putting it in the setter was kinda lazy. I'll move it up in the stack.